app/charts/column/columns-grouped.tsx

```diff
@@ -28,6 +28,9 @@ export const ErrorWhiskers = () => {
   const ref = useRef<SVGGElement>(null);
   const enableTransition = useTransitionStore((state) => state.enable);
   const transitionDuration = useTransitionStore((state) => state.duration);
+  // As xScale object is not re-created when its domain or range changes, we use this
+  // trick to force the re-rendering of the whiskers when the xScale changes.
+  const bandwidth = xScaleIn.bandwidth();
```

Comment:
Super nit: So if I understand correctly, if the domain and range change but the bandwidth happen to stay the same, we would not re-render right ? Will never happen I guess, but semantically we could also use directly xScaleIn.range() and xScaleIn.domain() in the useMemo deps right ?

Reply:
Here the problem is that we had `xScale` as a dependency, and this object was not updated when chart dimensions changed, because its range and domain were updated outside of `useMemo` in which it's created. I think it might also work to add xScaleIn.bandwidth() as dependency, but wondering if it would make it more clear? I think we still would need to have this comment (or re-create the whole `xScale` object when dimensions change) 🤔